Formula/jpeg-xl.rb

```diff
@@ -0,0 +1,57 @@
+class JpegXl < Formula
+  desc "New file format for still image compression"
+  homepage "https://jpeg.org/jpegxl/index.html"
+  url "https://gitlab.com/wg1/jpeg-xl/-/archive/v0.2/jpeg-xl-v0.2.tar.bz2"
+  sha256 "f0933c796f95ee905efa7a677367c0d57678b9587c2e967ea30d72e9405cca72"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "brotli"
+  depends_on "giflib"
+  depends_on "ilmbase"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "openexr"
+  depends_on "webp"
+
+  # These resources are versioned according to the script supplied with jpeg-xl to download the dependencies:
+  # https://gitlab.com/wg1/jpeg-xl/-/blob/master/deps.sh
+  resource "highway" do
+    revision = "311c183c9d96e69b123f61eedc21025dd27be000"
+    sha256 "40a4aaafbda61dd93803d8c830971f77d1b0b33c648ed29b5032f8798d293f21"
+    url "https://github.com/google/highway/archive/#{revision}.tar.gz"
```

Comment:
```suggestion
    url "https://github.com/google/highway.git",
        revision: "311c183c9d96e69b123f61eedc21025dd27be000"
```
Mind the number of spaces before `revision:`.